_docs/021-getting-started-gke.md

```diff
@@ -107,7 +106,8 @@ gcloud container clusters create $CLUSTER_NAME \
   --enable-autoscaling --min-nodes=1 --max-nodes=3 \
```

Comment:
since we are using auto-scaling maybe reduce the machine type to `n1-standard-2`, this seems to be more than plenty for trying riff out

Reply:
I'll try that thanks - This should also reduce the total CPU required from quota.